### PR TITLE
[Merged by Bors] - Updated product-config version

### PR DESCRIPTION
**Pull request by fhennig**
## Description

Just a version bump

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

**Review by teozkr — APPROVED** [review]

**Comment by teozkr** [comment]
Might want to note this in the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -74,7 +80,7 @@ All notable changes to this project will be documented in this file.
 - Unified `ClusterRef` type for referring to cluster objects ([#307]).
 
 ### Changed
-- BREAKING: kube 0.66 -> 0.68 ([#303]).
+- BREAKING: kube 0.66 -*Please add a description here. This will become the commit message of the merge request later.*> 0.68 ([#303]).
```
Comment (teozkr) [inline CHANGELOG.md:83]: Uhhh

**Comment by fhennig** [comment]
10 seconds too late :P haha

File: CHANGELOG.md (hunk 1 above)
Comment (fhennig) [reply CHANGELOG.md:83]: Oh lol

Comment (fhennig) [reply CHANGELOG.md:83]: I'm scrolling with my middle mouse button and accidentally pasted this :facepalm:

**Comment by fhennig** [comment]
you want to approve again?

**Review by teozkr — APPROVED** [review]

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**